src/qt/sendcoinsdialog.cpp

```diff
@@ -935,7 +943,8 @@ int SendConfirmationDialog::exec()
 {
     updateYesButton();
     countDownTimer.start(1000);
-    return QMessageBox::exec();
+    QMessageBox::exec();
+    return (buttonRole(clickedButton()) == QMessageBox::YesRole);
```

Comment:
Just noting that `clickedButton()` can return `nullptr` - in that case `buttonRole(nullptr)` returns `InvalidRole` so this is safe.

I wonder why not compare the `exec()` return agains `QMessageBox::Yes`?

Reply:
The YesRole button might not be Yes.

Reply:
And no, according to the Qt documentation, it can't return nullptr...

Reply:
But it can? https://doc.qt.io/qt-5/qmessagebox.html#clickedButton

Reply:
Oh because of `setEscapeButton`?

Reply:
Right